openedx/features/edly/tests/test_middleware.py

```diff
@@ -27,6 +27,14 @@ def setUp(self):
         self.request = RequestFactory().get('/')
         self.request.user = self.user
         self.request.site = SiteFactory()
+        self.site_config = SiteConfigurationFactory(
+            site=self.request.site,
+            enabled=True,
```

Comment:
we don't need to do this. `SiteConfigurationFactory` handles it by default.

Reply:
let me remove `enabled=True`